main.go

```diff
@@ -77,6 +79,7 @@ func main() {
 			"Enabling this will ensure there is only one active controller manager.")
 	flag.StringVar(&namespace, "namespace", "", "Argo CD repo namespace (default: argocd)")
 	flag.StringVar(&argocdRepoServer, "argocd-repo-server", "argocd-repo-server:8081", "Argo CD repo server address")
+	flag.IntVar(&argocdRepoServerTimeout, "argocd-repo-server-timeout", 60, "Argo CD repo server connection timeout")
```

Comment:
Unit is in seconds?   It's probably helpful to state the unit in the description.